### PR TITLE
Added The Sims

### PR DESCRIPTION
**Pull request by Zemogiter**
Script for the first game in the series. I've tried every wine version between 2.19-staging and 3.0 and the game crashed right before the loading screen no matter what I did. Seems to be some sort of a regression in wine. It also requires the "-skip_intro" line because the intro uses some really old codec wine does not support.

### CODE REVIEW

File: Applications/Games/The Sims/Local/script.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+include(["Engines", "Wine", "QuickScript", "LocalInstallerScript"]);
+
+new LocalInstallerScript()
+    .name("The Sims")
+    .editor("Maxis")
+    .applicationHomepage("https://www.ea.com/pl-pl/games/the-sims")
```
Comment (plata) [inline Applications/Games/The Sims/Local/script.js]: Please use the English version of the homepage.

File: Applications/Games/The Sims/Local/script.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+    .wineDistribution("staging")
+    .wineVersion("2.19-staging")
```
Comment (plata) [inline Applications/Games/The Sims/Local/script.js]: "2.19-staging" is not a valid `wineVersion` (should be "2.19").

**Comment by plata** [comment]
What shall we do with this? As you said, it's not usable currently.

**Comment by Zemogiter** [comment]
@plata What? I did not said it was unusable. The script works but it requires a specific version of wine.

**Comment by plata** [comment]
> I've tried every wine version between 2.19-staging and 3.0 and the game crashed right before the loading screen no matter what I did.

As the script is using 2.19-staging, I've assumed that the game crashes.

**Comment by Zemogiter** [comment]
Yes I meant the 2.19-staging is the last version of wine this game works on.

File: Applications/Games/The Sims/Local/script.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+new LocalInstallerScript()
+    .name("The Sims")
+    .editor("Maxis")
+    .applicationHomepage("http://www.thesims.com/")
+    .wineDistribution("staging")
+    .wineVersion("2.19")
+    .author("Zemogiter")
+    .category("Games")
+    .executable("Sims.exe", ["-skip_intro"])
+.go();
```
Comment (plata) [inline Applications/Games/The Sims/Local/script.js]: Please indent `.go();`.

**Pull request merged**